test/integration/BUILD.bazel

```diff
@@ -29,20 +31,20 @@ INTEGRATION_TEST_LIBRARIES = [
     "logging",  # Java package remapping in gapic.yaml.
     "redis",  # Has a gapic.yaml.
     "library",  # No gRPC service config.
-    "compute", # REGAPIC test.
+    "compute",  # REGAPIC test.
 ]
 
 # Keys must match the values in INTEGRATION_TEST_LIBRARIES above.
 API_GAPIC_TARGETS = {
-    "asset": "@com_google_googleapis//google/cloud/asset/v1:asset_java_gapic",
-    "credentials": "@com_google_googleapis//google/iam/credentials/v1:credentials_java_gapic",
+    "asset": ":asset_java_gapic",
+    "credentials": ":credentials_java_gapic",
     "iam": ":iam_java_gapic",  # Googleapis' LRO does not have a Java Gapic.
     "kms": ":kms_java_gapic",  # Local target because mixins are not rolled out yet.
     "pubsub": ":pubsub_java_gapic",
```

Comment:
Why are some tests coming from googleapis/googleapis-discovery and some are "native" to gapic-generator-java? (i.e. what is special about iam, kms and pubsub?)

Reply:
I added some custom changes to IAM, KMS, and PubSub to exercise edge cases that the `googleapis` target does not cover.